### PR TITLE
ARN-807 design content updates

### PR DESCRIPTION
**Pull request by Coupey**

### CODE REVIEW

File: common/templates/partials/header.njk [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
     <div class="moj-header__logo">
 
-      {{ renderMojCrest() }}
+      {{ renderMojCrest(crestColour, 40) }}
```
Comment (LucasCairns) [inline common/templates/partials/header.njk:10]: Not sure if I'm missing where `crestColour` is defined, if not I assume it's masking `undefined` which would then default in the macro?

**Review by LucasCairns — APPROVED** [review]
LGTM 👍

File: common/templates/partials/header.njk (hunk 1 above)
Comment (Coupey) [reply common/templates/partials/header.njk:10]: Yep - I'll sort this.

Comment (LucasCairns) [reply common/templates/partials/header.njk:10]: 👍

**Pull request merged**